packages/firestore/exp/src/api/database.ts

```diff
@@ -68,7 +69,7 @@ export const CACHE_SIZE_UNLIMITED = LRU_COLLECTION_DISABLED;
  * Do not call this constructor directly. Instead, use {@link getFirestore}.
  */
 export class FirebaseFirestore extends LiteFirestore {
-  readonly _queue = newAsyncQueue();
+  readonly _queue: AsyncQueue = newAsyncQueue();
```

Comment:
Somehow the `AsyncQueue` type is required to not break the APIExtractor toolchain.